Aliucord/src/main/java/com/aliucord/settings/Updater.java

```diff
@@ -122,6 +127,8 @@ public void onViewBound(View view) {
                                 try {
                                     updateAliucord(ctx);
                                     Utils.showToast(ctx, "Successfully updated Aliucord. Please restart Aliucord to load the update!");
+                                    showRestartButton = true;
+                                    Utils.mainThread.post(this::reRender);
```

Comment:
```suggestion
                                    getHeaderBar().findViewById(restartId).setVisibility(View.VISIBLE);
```

(and also add code to generate & add that id to the correct button)

Reply:
only reason i didn't do this was bc i thought you couldn't edit views on a different thread from the one they were created in

Reply:
post it to the main thread then :P but it should work i think idk just try lol